contracts/TokenHolder.sol

```diff
@@ -139,22 +121,8 @@ contract TokenHolder is MultiSigWallet {
         _;
     }
 
-    /**
-     * Requires that key was authorized. Key might be in authorized or
-     * revoked state.
-     */
-    modifier keyWasAuthorized(address _key)
-    {
-        AuthorizationStatus status = ephemeralKeys[_key].status;
-        require(
-            status != AuthorizationStatus.NOT_AUTHORIZED,
-            "Key was not authorized."
-        );
-        _;
-    }
-
     /** Requires that key was not authorized. */
-    modifier keyWasNotAuthorized(address _key)
+    modifier keyDoesNotExist(address _key)
```

Comment:
Better name can be: **keyShouldNotExist** 
Aligns better with submitAuthorizeSession

Reply:
`keyDoesNotExist` is clear; state either is or isn't and a function name therefore wouldn't contain "should"